pkg/terra/relay.go

```diff
@@ -32,11 +32,13 @@ type MsgEnqueuer interface {
 
 // CL Core OCR2 job spec RelayConfig member for Terra
 type RelayConfig struct {
-	ChainID string `json:"chainID"`
+	ChainID  string `json:"chainID"`  // required
+	NodeName string `json:"nodeName"` // optional
```

Comment:
Why is `NodeName` still optional here? b/c we'll still use the first node as the one if not specified, right?

Hmm, maybe we should also add the `chain-id` to the `node` data structure so we have that link and it can be verified. If the node is not explicitly specified we use a random (or first) one compatible with the specified chain.

If we don't know the `chain-id` for the `node`, and we leave the `spec.relay.nodeName` optional there is room for errors by misconfiguration.

Reply:
`chain-id` is already wired through, and the core PR updates the name-less case from _first_ to _random_ (still limited by chain id) to try to distribute any load by default: https://github.com/smartcontractkit/chainlink/pull/5853/files#diff-10e1587402f9125b6af71481bf4247044f5b411800c7367f091b7e4f91a64ad3R107

Reply:
Aha yeah, `chain-id` is already wired. Oh, this is nice then :) 

I'll document it in the job-spec example I'm updating...

Reply:
Please just extend the `// optional` comment with the same info here.

Reply:
Please do try to break this in testing though :wink: 
All of the Terra parts in core should be cleanly separated by chain id like with EVM: chainSet, nodes, txm, etc.